web_responsive/static/src/js/web_responsive.js

```diff
@@ -9,8 +9,9 @@ odoo.define('web_responsive', function(require) {
     var SearchView = require('web.SearchView');
     var core = require('web.core');
     var config = require('web.config');
-    var FieldOne2Many = core.form_widget_registry.get('one2many');
     var ViewManager = require('web.ViewManager');
+    var Chattter = require('mail.Chatter');
```

Comment:
If you do this, you must depend on mail. Check runbot, it's not loading this file.